tests/test.py

```diff
@@ -16,6 +21,62 @@
 class TestDSS(unittest.TestCase):
     def setUp(self):
         self.app = dss.create_app().app.test_client()
+        self.sre = re.compile("^assert(.+)Response")
+
+    def assertResponse(
+            self,
+            method: str,
+            path: str,
+            expected_code: int,
+            json_request_body: typing.Optional[dict]=None,
+            parse_response_as_json: bool=False,
+            **kwargs) -> typing.Tuple[wrappers.Response, str, typing.Optional[dict]]:
```

Comment:
Another typing newbie comment: can I just use `(x, y)` instead of `Tuple[x, y]`?